lib/kiev/logger.rb

```diff
@@ -13,6 +13,8 @@ class Logger
     def_delegators(*([:@logger] + ::Logger.instance_methods(false)))
 
     DEFAULT_EVENT_NAME = "log"
+    LOG_ERROR = "error"
```

Comment:
Log levels in this class, as I understand, come from `::Logger`. Its log levels are uppercased, check out `::Logger::SEV_LABEL`